tests/scripts/test_justfile_release_tag_quoting.py

```
@@ -0,0 +1,51 @@
+    marker = tmp_path / "interpolation-executed"
+    payload = f"$(touch {marker})"
+    command = [
+        "just",
+        *(argument.format(payload=payload) for argument in arguments),
+    ]
+    if not any("{payload}" in argument for argument in arguments):
+        command.append(payload)
+
+    environment = os.environ.copy()
+    environment["UV_CACHE_DIR"] = str(tmp_path / "uv-cache")
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)